lib/ansible/modules/windows/win_package.ps1

```diff
@@ -319,7 +319,11 @@ if ($state -eq "absent") {
                 $cleanup_artifacts += $local_path
             } elseif ($program_metadata.location_type -eq [LocationType]::Empty -and $program_metadata.msi -ne $true) {
                 # TODO validate the uninstall_string to see if there are extra args in there
-                $local_path = $program_metadata.uninstall_string
+                if ((@($program_metadata.uninstall_string).Count -eq 1) -and ($program_metadata.uninstall_string.StartsWith("`""))) {
```

Comment:
What's the reasoning behind the `@($program_metadata.uninstall_string).Count -eq 1` part?. I would also change the StartWith section to be `'"'` to not worry about escaping the double quotes. Same with the the Trim methods.